Felienne/hedy
# Add cyclic var definition error message #1408

**Description**

- Add error for cyclic var definitions: b is b + 1
- Remove duplications in lookup table. Now if a variable is present, it is hashed if it needs to be hashed or in plain text if it does not need to be hashed.
- Remove dead code from the transpiler: get_value function

**Fix for**

#1408 

**How to test**

- Run Hedy locally and go to any level >= 6. Type `b is b + 1` and see the error in English. Is the error text ok?
- Please translate the error in Dutch. You can add it as a comment or commit it directly.
- Have a thorough look at the adventure yaml files that are updated (because of the new cyclic error). Maybe there is something else with these that needs to change?

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
